openwisp_users/tests/test_admin.py

```diff
@@ -493,6 +493,44 @@ def test_organization_uuid_field(self):
         html = '<input type="text" name="name" value="default"'
         self.assertNotContains(response, html)
 
+    def test__action_active(self):
```

Comment:
there's a double underscore (`__`) in the test function name, can you remove one please?